src/html_json.js

```diff
@@ -94,29 +104,29 @@ async function fetchHTML(params, indices) {
   const results = new Map(await Promise.all(urls.map(async (url) => {
     log.info(`Reading HTML from: ${url}`);
 
-    let ret;
+    let resp;
     let body;
     try {
-      ret = await fetch(url, {
+      resp = await fetch(url, {
         headers: {
           'User-Agent': 'index-pipelines/html_json',
         },
         cache: 'no-store',
       });
-      body = await ret.text();
+      body = await resp.text();
     } catch (e) {
-      ret = {
+      resp = {
         ok: false,
         status: 500,
       };
       body = e.message;
     }
-    if (!ret.ok) {
+    if (!resp.ok) {
       const message = body < 100 ? body : `${body.substr(0, 100)}...`;
-      log.warn(`Fetching ${url} failed: statusCode: ${ret.status}, message: '${message}'`);
-      return [url, { error: { reason: message, status: ret.status } }];
+      log.warn(`Fetching ${url} failed: statusCode: ${resp.status}, message: '${message}'`);
+      return [url, { error: { reason: message, status: resp.status } }];
     }
-    return [url, { response: body }];
+    return [url, { body, headers: resp.headers }];
```

Comment:
Returning a copy of the headers would IMO be cleaner, i.e. 
`header: new Headers(resp.headers)`

Reply:
I see, where do I get a reference to the `Headers` class from? It is not defined in the `index.d.ts` file, only in `./fetch/headers.js`

Reply:
`const { fetch, Headers } = require('@adobe/helix-fetch');`